code/network/multi.cpp

```diff
@@ -1228,10 +1229,10 @@ void multi_do_frame()
 				}
 			} else if ( !(Player_ship->is_departing() ) ){
 				// if the rate limiting system says its ok
-				if(multi_oo_cirate_can_send()){
+//				if(multi_oo_cirate_can_send()){
 					// use the new method
-					multi_oo_send_control_info();
-				}				
+					multi_oo_send_control_info(); // Cyborg17 - Don't limit clients updating themselves.
+//				}				
```

Comment:
Is this really necessary? It basically makes object update rate limits irrelevant. And I'm a bit worried about what that will do to bandwidth on slower connections

Reply:
It makes it irrelevant for clients, yes.  I'll run some tests without it and check that it doesn't cause any slowdowns.

Reply:
It's not just the clients though, it greatly increases the bandwidth requirements for the server/standalone too. I'd feel a lot better about this with some hard data to back it up: 1) that it's actually necessary for the feature to work properly, and 2) that the bandwidth requirements for both server and client client are reasonable on a 120 FPS client with an upstream rate of no more than 512kbps.

Reply:
Testing after restoring, made only a slight decrease in interpolation accuracy.  I'll remove the change.